r2/r2/controllers/front.py

```diff
@@ -286,7 +286,8 @@ def GET_comments(
         infotext = None
         if request.GET.get('already_submitted'):
             submit_url = request.GET.get('submit_url') or article.url
-            resubmit_url = Link.resubmit_link(submit_url)
+            submit_title = request.GET.get('submit_title') or article.title
```

Comment:
Is it really appropriate to use `article.title` as the fallback? Wouldn't it be better to not pre-fill a title at all?